backend.py

```diff
@@ -217,6 +253,8 @@ def __init__(
         self._current_exposure_mode = auto_exposure
         self._preview_depth = preview_depth
 
+        self.init_device()
+
     def get_init_dict(self):
         return {
             "preview_depth": self._preview_depth,
```

Comment:
Make `record_pointcloud` session persistent
```suggestion
             "preview_depth": self._preview_depth,
             "record_pointcloud": self.record_pointcloud,
```